PhysicsTools/PatAlgos/python/cleaningLayer1/tauCleaner_cfi.py

```diff
@@ -8,7 +8,7 @@
         'tauID("decayModeFinding") > 0.5 &'
         ' tauID("byLooseCombinedIsolationDeltaBetaCorr3Hits") > 0.5 &'
         ' tauID("againstMuonTight3") > 0.5 &'
-        ' tauID("againstElectronVLooseMVA5") > 0.5'
+        ' tauID("againstElectronVLooseMVA6") > 0.5'
```

Comment:
Hi @slava77, 

sure I can take care of this within the next days and bubble it in along with other maintenance changes. It's on my list. 

Cheers,
Roger